app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ActiveRecord::Base
+  
+  has_many :comments, as: :commentable 
+  
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+  
+  validates :content, presence: true
+
+  default_scope -> { order(created_at: :desc) }
```

Comment:
Ahh I see - that would kill 2 birds with one stone, because I was wondering how exactly to have the posts be descending (get older as you scroll) and the replies to be ascending (get newer as you scroll).